pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -104,6 +104,30 @@ public void getActiveBrokers(@Suspended final AsyncResponse asyncResponse,
                 });
     }
 
+    @GET
+    @Path("/")
+    @ApiOperation(
+            value = "Get the list of active brokers (web service addresses) in the local cluster."
+                    + "If authorization is not enabled",
+            response = String.class,
+            responseContainer = "Set")
+    @ApiResponses(
+            value = {
+                    @ApiResponse(code = 401, message = "Authentication required"),
+                    @ApiResponse(code = 403, message = "This operation requires super-user access") })
+    public void getActiveBrokers(@Suspended final AsyncResponse asyncResponse) throws Exception {
+        validateSuperUserAccessAsync()
+                .thenCompose(__ -> pulsar().getLoadManager().get().getAvailableBrokersAsync())
+                .thenAccept(activeBrokers -> {
+                    LOG.info("[{}] Successfully to get active brokers", clientAppId());
+                    asyncResponse.resume(activeBrokers);
+                }).exceptionally(ex -> {
+                    LOG.error("[{}] Fail to get active brokers", clientAppId(), ex);
+                    resumeAsyncResponseExceptionally(asyncResponse, ex);
+                    return null;
+                });
```

Comment:
Most of the codes are duplicated to 
```java
getActiveBrokers(@Suspended final AsyncResponse asyncResponse,
                                 @PathParam("cluster") String cluster)
```
How about abstract a common method used by both the `getActiveBrokers(@Suspended final AsyncResponse asyncResponse, PathParam("cluster") String cluster)` and `getActiveBrokers(@Suspended final AsyncResponse asyncResponse)` ?

Reply:
Good suggestion, I removed duplicated code